index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Maybe we should implement some JavaScript features to include a button that has the functionality to reset our Tic Tac Toe game on our webpage.
```suggestion
<h2 hidden>Game over</h2>
<!--Insert JavaScript button which resets the game state back to the beginning-->
```